### PR TITLE
Bg/lwpmiopen 126 : Ignore running convbiasactive test on "gfx11" or "Navi3x"

### PR DESCRIPTION
**Pull request by bghimireamd**
* generate test cases based on GPU.

### CODE REVIEW

File: test/gtest/api_convbiasactiv.cpp [hunk 1]
```diff
@@ -168,22 +169,17 @@ TEST_P(ConvBiasActivFwdTest, DriveAPI)
     EXPECT_EQ(status, miopenStatusSuccess);
 }
 
+std::vector<CBATestCase> GetTestValues()
+{
+
+    std::vector<CBATestCase> ret;
+    if(get_handle().GetDeviceName() != "gfx1100")
```
Comment (JehandadKhan) [inline test/gtest/api_convbiasactiv.cpp]:
```suggestion
    if(miopen::StartsWith(get_handle().GetDeviceName(), "gfx11"))
```

File: test/gtest/api_convbiasactiv.cpp [hunk 2]
```diff
@@ -168,22 +169,17 @@ TEST_P(ConvBiasActivFwdTest, DriveAPI)
...
+        ret.push_back(CBATestCase{
+            16, 128, 16, 16, 128, 3, 3, 0, 0, 1, 1, 1, 1, miopenActivationRELU, miopenConvolution});
```
Comment (JehandadKhan) [inline test/gtest/api_convbiasactiv.cpp:177]:
```suggestion
            16, 128, 16, 16, 128, 3, 3, 0, 0, 1, 1, 1, 1, miopenActivationRELU, miopenConvolution});
    else 
    {
        GTEST_SKIP() << " Skipping fusion test on unsupported ASIC";
    }
```

**Review by JehandadKhan — CHANGES_REQUESTED** [review]

File: test/gtest/api_convbiasactiv.cpp (hunk 1 above)
Comment (junliume) [reply test/gtest/api_convbiasactiv.cpp]: And the title of the PR should be changed to "gfx11" or "Navi3x"

File: test/gtest/api_convbiasactiv.cpp (hunk 2 above)
Comment (bghimireamd) [reply test/gtest/api_convbiasactiv.cpp:177]: Added GTEST_SKIP()

File: test/gtest/api_convbiasactiv.cpp (hunk 1 above)
Comment (bghimireamd) [reply test/gtest/api_convbiasactiv.cpp]: changed the title as requested.

File: test/gtest/api_convbiasactiv.cpp [hunk 3]
```diff
@@ -59,6 +59,7 @@ struct CBATestCase
     }
 };
 
+GTEST_ALLOW_UNINSTANTIATED_PARAMETERIZED_TEST(ConvBiasActivFwdTest);
```
Comment (JehandadKhan) [inline test/gtest/api_convbiasactiv.cpp:62]: Why is this required?

**Review by JehandadKhan — APPROVED** [review]

File: test/gtest/api_convbiasactiv.cpp (hunk 3 above)
Comment (bghimireamd) [reply test/gtest/api_convbiasactiv.cpp:62]: `GTEST_ALLOW_UNINSTANTIATED_PARAMETERIZED_TEST ` allows empty list of parameterized test which is the case for Navi3x. If I do not do this it was giving error saying 

`Please use INSTANTIATE_TEST_SUITE_P to instantiate the tests, or GTEST_ALLOW_UNINSTANTIATED_PARAMETERIZED_TEST if the parameter list can be intentionally empty.`

**Review by junliume — APPROVED** [review]

**Review by junliume — APPROVED** [review]
